packages/admin/cms-admin/src/generator/future/utils/generateGridProps.ts

```diff
@@ -0,0 +1,56 @@
+import { IntrospectionQuery } from "graphql";
+
+import { GridConfig } from "../generator";
+import { getFilterGQLTypeString } from "./introspectionHelpers";
+
+export function hasGridPropBaseFilter({
```

Comment:
utils should only contain generic utils and not grid specific code.

- either move into generateGrid.ts (it's not that much code actually, I would even not necessarily create verbose functions)
- or add a file for grid specific helper functions
-